testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesTest.java

```diff
@@ -2283,6 +2282,45 @@ public void testExtendedClientPolicyIntefacesForCiba() throws Exception {
         assertEquals("Exception thrown intentionally", response.getErrorDescription());
     }
 
+    @Test
+    @EnableFeature(value = Profile.Feature.PAR, skipRestart = true)
+    @AuthServerContainerExclude({REMOTE, QUARKUS})
+    public void testExtendedClientPolicyIntefacesForPar() throws Exception {
```

Comment:
Similarly like for other PR, is it possible to either add this to the ParTest or create new test class? IMO it will be good if ClientPoliciesTest does not have any dependency on the preview features like Ciba or PAR.

Reply:
@mposolda I see, I will move this test from `ClientPoliciesTest` to `ParTest`.